src/webgpu/shader/validation/expression/call/builtin/unpack2x16float.spec.ts

```diff
@@ -0,0 +1,61 @@
+const kFn = 'unpack2x16float';
+export const description = `Validate ${kFn}`;
+
+import { makeTestGroup } from '../../../../../../common/framework/test_group.js';
+import { keysOf } from '../../../../../../common/util/data_tables.js';
+import { ShaderValidationTest } from '../../../shader_validation_test.js';
+
+const kArgCases = {
+  good: '(1u)',
```

Comment:
Please can you add a case for abstract-int? `1`
Same for the other tests

Reply:
Done. Added abstract_int for good args.